src/bots/mod.rs

```diff
@@ -50,6 +52,7 @@ impl Bots {
             Bots::AlwaysBetray => Box::new(AlwaysBetray::new()),
             Bots::GrimTrigger => Box::new(GrimTrigger::new()),
             Bots::Detective => Box::new(Detective::new()),
+            Bots::NguyenHanhNong => Box::new(nguyen_hanh_nong::NguyenHanhNong::new()),
```

Comment:
Can we change this to a use statement? At the top of the page, there are some examples of how this is done with the other bots :)